includes/Handler/class-mf2.php

```diff
@@ -595,6 +595,19 @@ protected function urls_match( $url1, $url2 ) {
 	protected function get_class_mapper() {
 		$class_mapper = array();
 
+		/*
+		 * These classes represent the source sending a webmention based on its display of a mention sent by the target.
+		 * This sets the type to update, indicating that the purpose of the webmention is to notify of an update to the original webmention.
+		 */
+		$class_mapper['comment']  = 'update';
```

Comment:
Can you explain that a bit more? I am not sure I can follow your thoughts!

Reply:
In each of these cases, it found the source link on a property indicating a marked up response. So, it isn't mentioning the target...at least that isn't the intention.

Reply:
But then I would vote for a complete remove of them, because otherwise these posts will never be updated!?!

Reply:
They are removed...in the receiver, after triggering an action. They never get added to the database. The mapper needs some way to tell the receiver to remove them...so using the comment_type.

Reply:
Sorry I was totally on the wrong path! Time for holidays!

So it is not (only) a parser problem, because it should have never sent the ping (because the url is a reaction and should not be pinged) and only because there where a ping, it was falsely parsed!

Makes also sense!